modules/crio_get.adoc

```diff
@@ -0,0 +1,117 @@
+{product-title} 3.10 and later, the docker service is no longer required. So you can now have
+CRI-O-only nodes in your {product-title} cluster.
+
+* Enabling CRI-O using a CRI-O enginer container is no longer supported. An
+rpm-based installation of CRI-O is required.
+
+The following procedure assumes you are installing {product-title} using Ansible
+inventory files, such as those described in
+link:https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html-single/installing_clusters/#install-config-configuring-inventory-file[Configuring Your Inventory File].
+
+[NOTE]
+====
+Do not set `/var/lib/docker` as a separate mount point for an {product-title} node
+using CRI-O as its container engine. When deploying a CRI-O node, the installer tries
+to make `/var/lib/docker` a symbolic link to `/var/lib/containers`. That action
+will fail because it won't be able to remove the existing `/var/lib/docker` to
+create the symbolic link.
+====
+
+. With the {product-title} Ansible playbooks installed, edit the appropriate
+inventory file to enable CRI-O. For example, when installing a single-node
+cluster on local host, edit the `hosts.localhost` file.
+
+. Locate CRI-O setting in your selected inventory file. To have the CRI-O
+container engine installed on your nodes during {product-title} installation, locate the
+following [OSEv3:vars] section of an Ansible inventory file:
++
+```
+[OSEv3:vars]
+...
+# Install and run cri-o.
+#openshift_use_crio=False
+#openshift_use_crio_only=False
+# The following two variables are used when openshift_use_crio is True
+# and cleans up after builds that pass through docker. When openshift_use_crio is True
+# these variables are set to the defaults shown. You may override them here.
+# NOTE: You will still need to tag crio nodes with your given label(s)!
+# Enable docker garbage collection when using cri-o
+#openshift_crio_enable_docker_gc=True
+# Node Selectors to run the garbage collection
+#openshift_crio_docker_gc_node_selector={'runtime': 'cri-o'}
+```
+
+. Enable CRI-O settings. You can decide to either enable CRI-O alone or CRI-O
+alongside Docker. The following settings allow CRI-O and docker as your node
+container engines and enables docker garbage collection on nodes with overlay2 storage:
+
++
+[NOTE]
+====
+To be able to build containers on CRI-O nodes, you must have the Docker
+container engine installed. If you want to have CRI-O-only nodes, you can do
+that and simply designate other nodes to do container builds.
+====
+
++
+```
+[OSEv3:vars]
+...
+openshift_use_crio=True
+openshift_use_crio_only=False
+openshift_crio_enable_docker_gc=True
+```
+
+The resulting {product-title} configuration will be running the CRI-O container engine on
+the nodes of your {product-title} installation.
+Use the oc command to check the status of the nodes and identify the nodes running CRI-O:
+
+```
+$ sudo oc get nodes -o wide
```

Comment:
@vikram-redhat Do we need `sudo`?

Reply:
I believe you are right. We do not. I'll remove it.